### PR TITLE
Refactor: remove clear all settings button in Image block

### LINKED ISSUE #2243: `Clear all settings` in BottomSheet options is visible only in Image block
**Describe the bug**
During testing [this PR](https://github.com/WordPress/gutenberg/pull/21365) I have noticed that not every block that has settings BottomSheet has also `Clear All Settings` button as the last action.

**Expected behavior**
Apply consistent approch to show `Clear all settings`

**Screenshots**

**Image block settings**

**Other mentioned block settings**
 Latest posts block settings |  Video block settings | Gallery block settings
--- | --- | ---
 |  | 

@iamthomasbishop what do you think ? Should we add `Clear all settings` to other block as well ?

### PR DESCRIPTION
**Pull request by jbinda**
Fixes #2243 

Please also refer to:
[Related gutenberg PR](https://github.com/WordPress/gutenberg/pull/22267)
[Related gutenberg-mobile issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2243)

To test:
See gutenberg PR for more details

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by etoledom** [comment]
It would be great to add a release-notes entry for this one, since it's a user-facing change.

**Comment by dratwas** [comment]
Hey @etoledom could you please check if the release-note is correct and approve if yes?

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ------
 * [**] Add support for changing overlay color settings in Cover block
 * Add enter/exit animation in FloatingToolbar
-
+* Remove `Clear all settings` button from Image block settings
```
Comment (etoledom) [inline RELEASE-NOTES.txt:5]: Actually, we should add the [*] indicating it's not an "hi pri" change for test

.
```suggestion
* [*] Remove `Clear all settings` button from Image block settings
```

**Comment by etoledom** [comment]
@dratwas - Is this still relevant?
I think we missed the chance to update the release notes already 🤔

**Comment by hypest** [comment]
Looks like the Gutenberg side PR has been merged in the past and the changes in this PR is no longer applicable so, closing.

**Pull request closed without merging**